src/scripts/views/left_column/LogoView.js

```diff
@@ -1,5 +1,5 @@
 define(['vendor/requirejs/text!templates/left_column/logo_nsidc.html',
-       'vendor/requirejs/text!templates/left_column/logo_acadis.html',
+       'vendor/requirejs/text!templates/left_column/logo_nsidc.html',
```

Comment:
Is the intention to render the NSIDC logo instead of the ACADIS logo in the left column? If the goal was to just remove the column, I think this string needs to be removed instead of changed, and logic for the `acadisLogoTemplate` should be removed.